packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -94,6 +96,8 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
                     odspLogger,
                     createNewSummary,
                     cacheAndTracker.epochTracker,
+                    fileEntry,
+                    this.hostPolicy.cacheCreateNewSummary ?? true,
```

Comment:
Set it to false by default and then once you follow with end-to-end test in next PR, then in same PR you can enable it as true by default.